Retarus/tests/test_retarus_events_forwarder.py

```diff
@@ -0,0 +1,68 @@
+import copy
+import os
+import time
+from datetime import datetime, timedelta, timezone
+from threading import Thread
+from unittest import mock
+from unittest.mock import Mock
+
+import orjson
+import pytest
+
+from retarus_modules.connector import RetarusConnector
+from tests.data import ORIGINAL_MESSAGE
```

Comment:
```suggestion
from tests.data import ORIGINAL_MESSAGE
```

You can replace this by the following fixture:

```python
pytest.fixture
def sample_retarus_message():
    yield "{\"direction\": \"OUTBOUND\", \"class\": \"EVENT\", \"version\": \"1.0\", \"type\": \"MTA\", \"ts\": \"2021-05-18 16:50:30 +0200\", \"host\": \"events.retarus.com\", \"customer\": \"45987FR\", \"metaData\": {}, \"sender\": \"utilisateur@mail.fr\", \"status\": \"ACCEPTED\", \"mimeId\": \"<d12b9brrfd3c89723ee5@STZE007.super.corp>\", \"rmxId\": \"20210518-32464-yvrfukcZEcd-0@out33.fg\", \"sourceIp\": \"255.255.255.1\", \"recipient\": \"recepient@mail.com\"}"
```

(raw message was taken from the intake format repository).